CIP-0013/CIP-0013.md

```diff
@@ -116,6 +120,31 @@ If, during a wallet or other application's development process, it is still only
 * any value for the first URI query argument;
 * any URI query argument beyond the first.
 
+### Handling Daedalus specific links
+
+Implementation should follow the [BIP-21](https://github.com/bitcoin/bips/blob/master/bip-0021.mediawiki) standard as is defined in [Specification](https://github.com/cardano-foundation/CIPs/blob/master/CIP-0013/CIP-0013.md#specification) section with `web+cardano:` replaced with specific custom protocol naming based on different Daedalus editions.
+
+Daedalus editions protocol naming:
+
+```
+cardanourn = "web+daedalus-(daedalusedition):" (paymentref | stakepoolref)
+daedalusedition = *(mainnet | mainnet_flight | testnet)
+
+Examples:
+
+Daeadalus Mainnet = web+daedalus-mainnet
+Daeadalus Flight = web+daedalus-mainnet_flight
```

Comment:
_Do_ you want the choice of "mainnet vs mainnet flight" to be made by the links themselves, rather than by the users?

E.g. if I only have Daedalus Flight installed, urls I find online with `web+daedalus-mainnet://` wouldn't work?

Reply:
> Do you want the choice of "mainnet vs mainnet flight" to be made by the links themselves, rather than by the users?

The network param would be used to preselect the correct application.
E.g. mainnet could be open up by Daedalus Mainnet, Daedalus Flight, Yoroi, AdaLite, etc.

I see Plutus apps offering users to chose themselves, but the network URL param preselects the default one.

Reply:
This is how we have envisioned it to work:

![base-url](https://user-images.githubusercontent.com/376611/137091771-910e89ca-acc9-424c-a33a-784cc09f5076.png)

Reply:
Buh ah, the `web+cardano:://` prefix, however, _would_ work with any wallet you have installed?

But where users, through PAB have the opportunity to specify the specific wallet/app they want to use, through the other prefixes?